core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
So we touched on this at TechTalk and people seemed to agree that in order to prevent users from shooting themselves in the foot by changing those values we'd just hardcode them and see if someone complains. I'll leave the timeout values in the conf though so those CAN be overridden